src/Hl7.Fhir.Specification.Tests/OutcomeReportingTest.cs

```diff
@@ -53,15 +53,15 @@ public void Setup()
         private OperationOutcome _report;
         private string _location;
 
-        [TestMethod]
+        [TestMethod, Ignore("The way we used hierarchy was broken, piloting another strategy now")]
```

Comment:
If the test is broken should we not fix it? Or otherwise remove it, when it is not applicable anymore?